src/api/spec/services/webui_controller_service/user_checker_spec.rb

```diff
@@ -28,7 +28,9 @@
     end
 
     context 'when the proxy authentication is enabled' do
-      let(:config) { { 'proxy_auth_mode' => :on } }
+      before do
+        stub_const('CONFIG', { proxy_auth_mode: :on }.with_indifferent_access)
```

Comment:
This looks like we test the old `CONFIG['proxy_auth_mode']` that we want to replace. Is that intended? Should we also test the new stuff?

Reply:
We don't want to replace the setting, we want to replace checking it in the code.

Reply:
applied your suggestion @dmarcoux